src/database/type.ts

```diff
@@ -79,7 +79,10 @@ export class PersonIdentifier extends Identifier {
 
 export enum PreDefinedVirtualGroupNames {
     friends = '_default_friends_group_',
+    followers = 'database_group_followers_name',
```

Comment:
should it called this i18n name instead of a better semantic name?

Reply:
Yes. It's a bit wried using different naming style. I grab the key name of i18n messages directly as group id in order to simply the name mapping logic in `resolveSpecialGroupName.ts`. Otherwise we should hard code the mapping rules like what we did on `PreDefinedVirtualGroupNames .friends` field.

Reply:
Here is the hard coded mapping logic:

```typescript
    if (group.groupName === PreDefinedVirtualGroupNames.friends) {
        return geti18nString('database_group_friends_default_name', owner)
    }
```

Reply:
I think it's better to have a hard coded mapping because they are speical groups. and won't be too much.

Reply:
OK I changed it!